lib/sign-in.js

```diff
@@ -0,0 +1,39 @@
+module.exports = signIn
+
+var Promise = require('../utils/promise')
+
+var internals = module.exports.internals = {}
+internals.deserialise = require('../utils/deserialise')
+internals.request = require('../utils/request')
+internals.saveSession = require('../utils/save-session')
+internals.serialise = require('../utils/serialise')
```

Comment:
we usually make blocks for declarations of variables, the order is

1. node-internal requires like `var fs = require('fs')`
2. requires from package dependencies like `var test = require('tape')`
3. requires from within the module like `var Promise = require('../utils/promise')`
4. everything else :)

In each block, we sort the lines alphabetically